src/preamble.js

```diff
@@ -1470,6 +1480,10 @@ function setF64(ptr, value) {
 
 #endif // USE_PTHREADS
 
+function getTotalMemory() {
```

Comment:
This was added so that we can query from inside asm.js `sbrk` to outside world what the current `TOTAL_MEMORY` is, if it got resized. I pondered on making the asm.js module own the `TOTAL_MEMORY` variable as well, but that is not necessary, at least not yet. (Not sure if we get around to doing memory resizing with pthreads support at some point)